static/test/javascripts/spec/common/weather.spec.js

```diff
@@ -14,10 +14,10 @@ define([
     Injector
     ) {
 
-    return  new Injector()
+    return new Injector()
         .store('common/utils/config')
         .require(['common/modules/weather', 'mocks'], function (sut, mocks) {
-            ddescribe('Weather component', function () {
+            describe('Weather component', function () {
```

Comment:
this is actually valid in jasmine code; it means 'don't run this suite'. however, it seems to suppress this and all future ones - might be due to the way we're running them, need to investigate